src/swarm-engine/Swarm/Log.hs

```diff
@@ -22,19 +23,34 @@ module Swarm.Log (
   logToText,
 ) where
 
-import Control.Lens (makeLenses, view)
-import Data.Aeson (FromJSON, ToJSON)
+import Control.Lens (makeLenses, over, view, _head)
+import Data.Aeson
+import Data.Char (toLower)
 import Data.Foldable (toList)
 import Data.Text (Text)
 import GHC.Generics (Generic)
+import Servant.Docs qualified as SD
 import Swarm.Game.Location (Location)
-import Swarm.Game.Tick (TickNumber)
+import Swarm.Game.Tick (TickNumber (..))
 import Swarm.Game.Universe (Cosmic)
+import Swarm.Util.JSON (optionsMinimize)
+
+enumJsonOptions :: Options
+enumJsonOptions =
```

Comment:
I think this already exists as `Swarm.Util.JSON.optionsUntagged`.

Reply:
Good point, fixed. 👍